code/modules/vehicles/mecha/makeshift/lockermech.dm

```diff
@@ -0,0 +1,56 @@
+/obj/vehicle/sealed/mecha/makeshift
+	desc = "A locker with stolen wires, struts, electronics and airlock servos crudely assembled into something that resembles the functions of a mech."
+	name = "Locker Mech"
+	icon = 'icons/mecha/lockermech.dmi'
+	icon_state = "lockermech"
+	base_icon_state = "lockermech"
+	max_integrity = 100 //its made of scraps
+	lights_power = 5
+	movedelay = 4 //Same speed as a ripley, for now.
```

Comment:
Ripley MK2 akshually...

Reply:
didn't touch the comments lmao, this is so ancient the mk2 probably didn't exist yet

Reply:
yeah, figured that, also fun fact: the MK2 is literally the old ripley
please change it through, it's inconsistency is annoying me :suffering"